test/grisp_connect_api_SUITE.erl

```diff
@@ -49,16 +47,66 @@ init_per_testcase(TestCase, Config) ->
             ?assertEqual(ok, wait_connection()),
             grisp_connect_test_server:listen()
     end,
-    Config.
+    [{apps, Apps} | Config].
 
+end_per_testcase(TestCase, Config)
+    when TestCase =:= bad_client_version_test;
+         TestCase =:= bad_server_version_test  ->
+      Config;
 end_per_testcase(_, Config) ->
     ok = application:stop(grisp_connect),
     grisp_connect_test_server:wait_disconnection(),
     ?assertEqual([], flush()),
+    grisp_connect_test_server:stop(proplists:get_value(apps, Config)),
```

Comment:
Just a note: You can use the common test macro `?config(apps, Config)` here instead of `proplists:get_value/2`.

Reply:
The `?config` macro is deprecated: https://www.erlang.org/doc/apps/common_test/ct.html

Reply:
I didn't know that. (Mostly still working with OTP 23 in DAB :)